clicache/src/Cache.cpp

```diff
@@ -99,7 +99,7 @@ namespace Apache
         {
           auto nativeptr = std::dynamic_pointer_cast<InternalCacheTransactionManager2PC>(
             m_nativeptr->get()->getCacheTransactionManager());
```

Comment:
I thought this was now supposed to return a ref not a shared_ptr?

Reply:
That seems reasonable to me.  But, that's a change to `Cache` interface and I assumed scope of this change was only `CacheTransactionManager`.